pai-management/src/end-to-end-test/dockerfile

```diff
@@ -17,6 +17,7 @@
 
 FROM hadoop-run
 
+RUN rm -rf /var/lib/apt/lists/*
 RUN apt-get -y update && \
```

Comment:
Why not to fix this problem in hadoop-run image? Is this one required by other images inherited from hadoop-run?

Reply:
I believe we should fix it in both.
But the Dockerfile of hadoop-run was generated on the fly, in this case, we'd better put some redundancy here for protecting.

Reply:
Yes, I think so, other images inherited from hadoop-run will need this.